src/runtime_src/core/common/query_requests.h

```diff
@@ -3129,6 +3130,19 @@ struct clk_scaling_info : request
   get(const device*) const = 0;
 };
 
+struct xgq_scaling_configure : request
```

Comment:
We have generally been bad at document this query request structs.  At some point we should sweep and document them all, but since this one is new, please add comments as to what it actually returns and what xgq_scaling_configure means.  Please spread the words that documentations of query structs is required.

Reply:
Hi @stsoe thank you for the review. I have added comments in the code as per suggestions.
Thank you.